tests/system/test_retry.py

```diff
@@ -59,7 +59,8 @@ async def test_retry_bubble_async(async_echo):
 
     @pytest.mark.asyncio
     async def test_method_async_wrapper_for_async_client(async_echo):
-        with pytest.raises(exceptions.NotFound):
+        # TODO(https://github.com/googleapis/gapic-generator-python/issues/2152): Add test for retrying LRO.
```

Comment:
Add link to LRO feature request: https://github.com/googleapis/gapic-generator-python/issues/2170